gatecrasher.md

```diff
@@ -0,0 +1,41 @@
+# Gatecrasher
+Screencast
+
+![alt gatecrasher_pic](https://github.com/foundersandcoders/case-studies/blob/gatecrasher/assets/GateCrasher%20.gif)
+
+Client Quote
```

Comment:
Maybe delete this section?

The "client quote" is for the client's reaction after seeing the app that you made